### ISSUE TITLE
help with finetune process

### ISSUE THREAD

**Issue by daddym85** [tensorflow/tensorflow#5540]
Hi there,
I have run the finetune_inception_v3_on_flowers.sh slim script example, and I have obtained the graph.pbtxt file. How can I use this file in a Python script for image classification? (e.g., https://github.com/tensorflow/tensorflow/blob/master/tensorflow/models/image/imagenet/classify_image.py)
Thanks.

**Comment by sherrym** [comment]
Hi @daddym85 ,

You should be able to follow instructions here:

https://github.com/tensorflow/models/blob/master/slim/README.md

Thanks,
Sherry

**Comment by daddym85** [comment]
Hi there,
I have run the command:

python train_image_classifier.py \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=flowers \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=inception_v3 \
  --checkpoint_path=${PRETRAINED_CHECKPOINT_DIR}/inception_v3.ckpt \
  --checkpoint_exclude_scopes=InceptionV3/Logits,InceptionV3/AuxLogits \
  --trainable_scopes=InceptionV3/Logits,InceptionV3/AuxLogits \
  --learning_rate=0.01 \
  --learning_rate_decay_type=fixed \
  --save_interval_secs=60 \
  --save_summaries_secs=60 \
  --log_every_n_steps=100 \
  --weight_decay=0.00004

to generate model.ckpt-XXXX file.
How can I use such a file in the freeze_graph script in order to build a pb file?
Thanks.

**Comment by girving** [comment]
Questions like this are better asked on StackOverflow.  Github issues are for bug reports and feature requests.

**Comment by gzygmontson13** [comment]
we explored the role of managers in transforming formal learning processes. Our latest benchmark report, Unlocking Potential, shows that this group of employees is the key to unlocking more effective learning.
We know from our ongoing research with over 35,000 learners that 78% find support from their manager essential or very useful and 48% say that [it](https://www.2gna.com/) is their manager or director who would most influence them to learn online.
As well as focusing on the role of managers in supporting learning, our data provides insights into the tactics that are helping organisations succeed at fine-tuning their processes.
Fine-tuning is the process investors and investment professionals use to make small modifications or improvements to investment portfolios. It can be executed using different strategies, such as technical analysis, either manually or automatically using new technology.
BERT relies on massive compute for pre-training ( 4 days on 4 to 16 Cloud TPUs; pre-training on 8 GPUs would take 40–70 days i.e. is not feasible. BERT fine tuning tasks also require huge amounts of processing power, which makes it less attractive and practical for all but very specific tasks.
